frontend/src/plugins/impl/chat/chat-ui.tsx

```diff
@@ -253,6 +253,18 @@ export const Chatbot: React.FC<Props> = (props) => {
     });
   }, [messages.length, scrollContainerRef]);
 
+  const resetInput = () => {
+    // Clear input immediately by directly manipulating the editor
+    // There is some delay if we use setInput("") only
+    if (codeMirrorInputRef.current?.view) {
```

Comment:
nit: might as well make it `const view = codeMirrorInputRef.current?.view` and simplify below